documentation/modules/ROOT/pages/modeling-guidelines.adoc

```diff
@@ -94,6 +94,28 @@ xref:namespaces.adoc#namespaces-and-versions[Namespaces and Versions] for more d
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 ----
 
+[[declaring-namespaces]]
+=== Declaring Namespaces
+
+NOTE: Available from 2.2.0 SAMM version
```

Comment:
Although I like the idea of highlighting the meta model version that introduces a certain construct, this is currently not done anywhere else, so it would be inconsistent with the rest of the spec. Therefore I'd remove this note.

Reply:
So, okay
I will remove it

Thanks!